addons/sys_core/fnc_vehicleCrewPFH.sqf

```diff
@@ -16,15 +17,51 @@
 #include "script_component.hpp"
 
 private _vehicle = vehicle acre_player;
+private "_vehicleIntercom";
```

Comment:
This line shouldn't be neccesary as `_vehicleIntercom` is never used in this scope.